packages/web-client/tests/integration/components/card-pay/withdrawal-workflow/transaction-status-test.ts

```diff
@@ -59,30 +64,29 @@ module(
         .containsText(
           `Bridge tokens from ${c.layer2.fullName} to ${c.layer1.fullName}`
         );
-      assert.dom(`[data-test-bridge-explorer-button]`).doesNotExist();
+      assert
+        .dom(`[data-test-bridge-explorer-button]`)
+        .hasAttribute('href', /relay$/);
     });
 
     test('It completes when the bridged transaction completes', async function (assert) {
       assert.ok(onComplete.notCalled);
 
-      let layer1Service = this.owner.lookup('service:layer1-network')
-        .strategy as Layer1TestWeb3Strategy;
-
-      layer1Service.test__simulateBridged('0xbridged');
+      layer2Service.test__simulateBridgedToLayer1();
 
       await settled();
 
-      assert
-        .dom('[data-test-action-card-title-icon-name="success-bordered"]')
-        .exists();
-
       assert
         .dom(`[data-test-token-bridge-step="1"][data-test-completed]`)
         .exists();
       assert
         .dom(`[data-test-bridge-explorer-button]`)
         .hasAttribute('href', /relay$/);
 
+      assert
```

Comment:
quite minor but the preceding assertion can be removed as it’s now covered in the preceding test